packages/main/src/TableRow.js

```diff
@@ -205,6 +351,12 @@ class TableRow extends UI5Element {
 	getNormilzedTextContent(textContent) {
 		return textContent.replace(/[\n\r\t]/g, "").trim();
 	}
+
+	static async onDefine() {
+		await Promise.all([
+			fetchI18nBundle("@ui5/webcomponents"),
```

Comment:
just 
await fetchI18nBundle("@ui5/webcomponents"),

use Promise.all for multiple async tasks